### ISSUE TITLE
Rename address env property to account_id

### ISSUE THREAD

**Issue by kigawas** [paritytech/ink#266]
```rust
    impl_get_property_for! {
        /// Returns the address of the caller of the executed contract.
        fn caller<Caller>() -> T::AccountId;
        /// Returns the transferred balance for the contract execution.
        fn transferred_balance<TransferredBalance>() -> T::Balance;
        /// Returns the current price for gas.
        fn gas_price<GasPrice>() -> T::Balance;
        /// Returns the amount of gas left for the contract execution.
        fn gas_left<GasLeft>() -> T::Balance;
        /// Returns the current block time in milliseconds.
        fn now_in_ms<NowInMs>() -> T::Moment;
        /// Returns the address of the executed contract.
        /* rename? */ fn address<Address>() -> T::AccountId;
        /// Returns the balance of the executed contract.
        fn balance<Balance>() -> T::Balance;
        /// Returns the current rent allowance for the executed contract.
        fn rent_allowance<RentAllowance>() -> T::Balance;
        /// Returns the current block number.
        fn block_number<BlockNumber>() -> T::BlockNumber;
        /// Returns the minimum balance of the executed contract.
        fn minimum_balance<MinimumBalance>() -> T::Balance;
    }

```

#66

**Comment by Robbepop** [comment]
I think that's a good idea. I also thought about this but didn't do it to match the old interface.

**Comment by kigawas** [comment]
Maybe just add a new function and label the old as deprecated

**Comment by Robbepop** [comment]
This is implemented in #312 and can be closed once the PR has been merged.
